docs/self-managed/setup/deploy/amazon/amazon-eks/eks-helm.md

```diff
@@ -181,139 +223,424 @@ spec:
+
+If you prefer not to use an external PostgreSQL service, you can switch to the internal PostgreSQL deployment. In this case, you will need to configure the Helm chart as follows and remove certain configurations related to the external database and service account:
+
+<details>
+<summary>Show configuration changes to disable external database usage</summary>
+
+```yaml
+identityKeycloak:
+  postgresql:
+    enabled: true
+
+  # Remove external database configuration
+  # externalDatabase:
+  #   ...
+
+  # Remove service account and annotations
+  # serviceAccount:
+  #   ...
+
+  # Remove extra environment variables for external database driver
+  # extraEnvVars:
+  #   ...
+
+webModeler:
+  # Remove this part
+
+  # restapi:
+  #     externalDatabase:
+  #         url: jdbc:aws-wrapper:postgresql://${DB_HOST}:5432/${DB_WEBMODELER_NAME}
+  #         user: ${DB_WEBMODELER_USERNAME}
+  #         existingSecret: webmodeler-postgres-secret
+  #         existingSecretPasswordKey: password
+
+identity:
+  # Remove this part
+
+  # externalDatabase:
+  #     enabled: true
+  #     host: ${DB_HOST}
+  #     port: 5432
+  #     username: ${DB_IDENTITY_USERNAME}
+  #     database: ${DB_IDENTITY_NAME}
+  #     existingSecret: identity-postgres-secret
+  #     existingSecretPasswordKey: password
+```
+
+</details>
+
+##### Fill your deployment with actual values
+
+Once you've prepared the `values.yml` file, use the `envsubst` command to substitute the environment variables with their actual values. Run the following command:
+
+```bash
+# generate the final values
+envsubst < values.yml > generated-values.yml
+
+# print the result
+cat generated-values.yml
+```
+
+Starting from **Camunda 8.6**, you need to store various passwords in a Kubernetes secret, which will be consumed by the Helm chart. Below is an example of how to set up the required secret:
+
+You can use `openssl` to generate random secrets and store them in environment variables:
+
+```bash reference
+https://github.com/camunda/camunda-tf-eks-module/blob/feature/opensearch-doc/examples/camunda-8.6/procedure/generate-passwords.sh
+```
+
+Next, use these environment variables in the `kubectl` command to create the secret.
+
+Note:
+
+- The values for `postgres-password` and `password` are not required if you are using an external database. If you choose not to use an external database, please provide those values.
+- The `smtp-password` should be replaced with the appropriate external value.
+
+```bash reference
+https://github.com/camunda/camunda-tf-eks-module/blob/feature/opensearch-doc/examples/camunda-8.6/procedure/create-identity-secret.sh
+```
+
+#### 3. Install Camunda 8 using Helm
+
+Now that the `generated-values.yml` is ready, you can install Camunda 8 using Helm. Here's the command:
+
+```bash reference
+https://github.com/camunda/camunda-tf-eks-module/blob/feature/opensearch-doc/examples/camunda-8.6/procedure/install-chart.sh
```

Comment:
not sure whether we should suggest SNAPSHOT as default for next but we can.

it would require the following adjustment for the helm chart to be used

```
helm upgrade --install camunda oci://ghcr.io/camunda/helm/camunda-platform --version 0.0.0-snapshot-alpha
```

Reply:
essentially, the version defined a bit above would be `0.0.0-snapshot-alpha` doesn't change ever.

Reply:
`SNAPSHOT` might fail if people are following it ... at least for me it's failing right now :D

Reply:
just saying that it would require some adjustments to as with just `SNAPSHOT` it can't be followed 1:1